src/View/Sidebar.vala

```diff
@@ -982,9 +982,9 @@ public class Marlin.Sidebar : Marlin.AbstractSidebar {
                         format_size (fs_capacity)
                     );
 
-                    var tooltip = "%s\n<span weight=\"600\" size=\"smaller\" alpha=\"75%\">%s</span>".printf (
+                    var tooltip = "%s\n%s".printf (
                         used_string,
-                        size_string
+                        Granite.TOOLTIP_SECONDARY_TEXT_MARKUP.printf (size_string)
```

Comment:
```suggestion
                    var tooltip = ("%s\n"+Granite.TOOLTIP_SECONDARY_TEXT_MARKUP).printf (
                        used_string,
                        size_string
```
this way it is even more efficient as:
 * The `"%s\n"+Granite.TOOLTIP_SECONDARY_TEXT_MARKUP` string is created at compile time
 * we are only allocating the string once (only one printf)